src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -162,7 +162,7 @@ def make_artifacts(
 )
 def make_results(
     output_dir: str,
-    version: str,
+    label_version: str,
```

Comment:
Why this change?

Reply:
Not sure why this is showing up. This is from a bugfix that I rebased on